src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -201,9 +201,11 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
 
     # Resource group Creation
     if resource_group_exists(cmd.cli_ctx, resource_group_name, subscription_id) is False:
-        resource_group_params = {'location': location}
+        from azure.cli.core.profiles import ResourceType
+        ResourceGroup = cmd.get_models('ResourceGroup', resource_type=ResourceType.MGMT_RESOURCE_RESOURCES)
+        parameters = ResourceGroup(location=location)
         try:
-            resourceClient.resource_groups.create_or_update(resource_group_name, resource_group_params)
+            resourceClient.resource_groups.create_or_update(resource_group_name, parameters)
```

Comment:
We are also using the resourceClient here: https://github.com/Azure/azure-cli-extensions/blob/1010d2af2fe867a9b361d30db8f3f44e4868816a/src/connectedk8s/azext_connectedk8s/_utils.py#L54

Reply:
Thanks for your reminder~ In fact, I know it. However, there is no breaking change for this operation ([code link](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/resources/azure-mgmt-resource/azure/mgmt/resource/resources/v2020_10_01/operations/_providers_operations.py#L367)), so no additional compatibility logic is required.